pyteal/ast/abi/tuple_test.py

```diff
@@ -662,7 +662,7 @@ def test_Tuple_set():
     with pytest.raises(pt.TealInputError):
         tupleValue.set(uint8, uint16, uint32, uint32)
 
-    with pytest.raises(pt.TealInputError):
+    with pytest.raises(pt.TealInputError, match="at 1: uint32 != uint16"):
```

Comment:
if you take the change of error msg above, also take this change
```suggestion
    with pytest.raises(pt.TealInputError, match="at index=1: uint32 != uint16"):
```